cmd/dedupe/dedupe.go

```diff
@@ -37,7 +37,7 @@ Opendrive) that can have duplicate file names. It can be run on wrapping backend
 (e.g. crypt) if they wrap a backend which supports duplicate file
 names.
 
-However if --by-hash is passed in then dedupe will find files with
+However if `--by-hash` is passed in then dedupe will find files with
```

Comment:
This will need to be like this as you can't put backquotes inside backquoted strings

```go
However if  `+"`--by-hash`"+` is passed
```

Reply:
Thanks!